xml/tuning-how.xml

```diff
@@ -0,0 +1,227 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+    %entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="tuning-how">
+ <title>General Notes on System Tuning</title>
+ <info>
+  <dm:docmanager xmlns:dm="urn:x-suse:ns:docmanager">
+   <dm:editurl>https://github.com/SUSE/doc-ses/edit/master/xml/</dm:editurl>
+   <dm:translation>yes</dm:translation>
+   <dm:release>SES 7</dm:release>
+  </dm:docmanager>
+ </info>
+ <para>
+    This manual discusses how to find the reasons for performance problems
+    and provides means to solve these problems. Before you start tuning your
+    system, you should make sure you have ruled out common problems and have
+    found the cause for the problem. You should also have a detailed plan on
+    how to tune the system, because applying random tuning tips often will
+    not help and could make things worse.
+   </para>
+ <procedure>
+  <title>General Approach When Tuning a System</title>
+  <step>
+   <para>
+    Specify the problem that needs to be solved.
+   </para>
+  </step>
+  <step>
+   <para>
+    In case the degradation is new, identify any recent changes to the
+    system.
+   </para>
+  </step>
+  <step xml:id="tuning-basics-whyproblem">
+   <para>
+    Identify why the issue is considered a performance problem.
+   </para>
+  </step>
+  <step>
+   <para>
+    Specify a metric that can be used to analyze performance. This metric
+    could for example be latency, throughput, the maximum number of
+    users that are simultaneously logged in, or the maximum number of active users.
+   </para>
+  </step>
+  <step>
+   <para>
+    Measure current performance using the metric from the previous step.
+   </para>
+  </step>
+  <step>
+   <para>
+    Identify the subsystem(s) where the application is spending the most
+    time.
+   </para>
+  </step>
+  <step>
+   <para/>
+   <substeps performance="required">
+    <step>
+     <para>
+      Monitor the system and/or the application.
+     </para>
+    </step>
+    <step>
+     <para>
+      Analyze the data, categorize where time is being spent.
+     </para>
+    </step>
+   </substeps>
+  </step>
+  <step>
+   <para>
+    Tune the subsystem identified in the previous step.
+   </para>
+  </step>
+  <step>
+   <para>
+    Remeasure the current performance without monitoring using the same
```

Comment:
Is "remeasure" a real word?

I may be missing something but how does one measure the performance without monitoring?

Reply:
It is, actually :joy: but I had the same thought